applications/jupyter/outputs.tf

```diff
@@ -14,7 +14,7 @@
 
 output "jupyterhub_uri" {
   description = "JupyterHub Endpoint to access user interface. In case of private IP, consider port-forwarding."
-  value       = "http://${module.jupyterhub.jupyterhub_uri}"
+  value       = module.jupyterhub.jupyterhub_uri != "" ? "http://${module.jupyterhub.jupyterhub_uri}" : local.jupyterhub_default_uri
```

Comment:
Can you please update RAG and Jupyter READMEs with how to use this? We need to mention port forwarding before logging in with dummy credentials.

Reply:
will be taken care of in https://github.com/GoogleCloudPlatform/ai-on-gke/pull/384